typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -189,6 +187,7 @@ export function deserializeRedBlackTree<Key, Value>(
   const nodes = tree_nodes[0];
   const freeNodes = tree_nodes[1];
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
why is this needed?

Reply:
It treats the underscore as a used variable for some reason....

Reply:
lol. I think you can do
`for (const [index, [key, ]] of node.entries())`